src/nvim/mbyte.c

```diff
@@ -1840,7 +1840,7 @@ int mb_off_next(const char_u *base, const char_u *p)
 /// Return the offset from "p" to the last byte of the character it points
 /// into.  Can start anywhere in a stream of bytes.
 /// Composing characters are not included.
```

Comment:
```suggestion
/// Return the offset from `p_in` to the last byte of the codepoint it points
/// to.  Can start anywhere in a stream of bytes.
/// Note: Unlike `utf_tail_off`, individual codepoints of composing characters 
/// are counted separately.
```
(and similarly for `utf_cp_head_off`, which I can't comment on).